scipy/spatial/qhull.pyx

```diff
@@ -2150,6 +2150,20 @@ cdef int _get_delaunay_info(DelaunayInfo_t *info,
     return 0
 
 
+def _has_nan(points):
+    """
+    has_nan(points)
+
+    Checks whether a point in an array of points has a NaN value.
+
+    .. versionadded:: 0.17.0
+
+    """
+    for point in points:
```

Comment:
For loop over potentially big arrays is slow --- this routine is similar to one-liner `np.isnan(points).all()`. Do also `inf` in the data crash Qhull? If so, it should be `not np.isfinite(points).all()`